manifests/m/Microsoft/WindowsSDK/10/0/2104/10.0.20348.0/Microsoft.WindowsSDK.10.0.2104.installer.yaml

```diff
@@ -0,0 +1,23 @@
+# Created using wingetcreate 1.5.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Microsoft.WindowsSDK.10.0.2104

```

Comment:
Hmmm, it look like you used 5 code. I don't know what is my 5 code. I know that one is 2104